src/application/components/Modals/ClientNotFoundModal.tsx

```diff
@@ -1,35 +1,33 @@
-import { Button } from "./Button";
-import { ButtonGroup } from "./ButtonGroup";
-import { Disclosure } from "./Disclosure";
-import { GitHubIssueLink, SECTIONS, LABELS } from "./GitHubIssueLink";
-import { Modal } from "./Modal";
+import { Button } from "../Button";
+import { Disclosure } from "../Disclosure";
+import { ExternalLink } from "../ExternalLink";
+import { GitHubIssueLink, SECTIONS, LABELS } from "../GitHubIssueLink";
+import { Modal } from "../Modal";
 import IconGitHubSolid from "@apollo/icons/small/IconGitHubSolid.svg";
 
 interface ClientNotFoundModalProps {
   open: boolean;
-  onClose: () => void;
   onRetry: () => void;
 }
```

Comment:
This modal cannot be closed anymore - before, closing it would leave the user with a somewhat defunct UI, so it never made much sense to me. Going this step massively simplifies logic.